core/client/app/templates/components/gh-nav-menu.hbs

```diff
@@ -20,22 +20,26 @@
         <button class="gh-nav-search-button"><i class="icon-search"></i><span class="sr-only">Search</span></button>
     </section>
     <ul class="gh-nav-list gh-nav-main">
-        {{!<li><i class="icon-dash"></i>Dashboard</li>}}
-        <li>{{#link-to "editor.new" classNames="gh-nav-main-editor"}}<i class="icon-pen"></i>New Post{{/link-to}}</li>
-        <li>{{#link-to "posts" classNames="gh-nav-main-content"}}<i class="icon-content"></i>Content{{/link-to}}</li>
-        {{!<li><a href="#"><i class="icon-user"></i>My Posts</a></li>}}
-        <li>{{#link-to "team" classNames="gh-nav-main-users"}}<i class="icon-team"></i>Team{{/link-to}}</li>
-        {{!<li><a href="#"><i class="icon-idea"></i>Ideas</a></li>}}
+        {{#gh-nav-list onClick="closeMenus"}}
+            {{!<li><i class="icon-dash"></i>Dashboard</li>}}
+            <li>{{#link-to "editor.new" classNames="gh-nav-main-editor"}}<i class="icon-pen"></i>New Post{{/link-to}}</li>
+            <li>{{#link-to "posts" classNames="gh-nav-main-content"}}<i class="icon-content"></i>Content{{/link-to}}</li>
+            {{!<li><a href="#"><i class="icon-user"></i>My Posts</a></li>}}
+            <li>{{#link-to "team" classNames="gh-nav-main-users"}}<i class="icon-team"></i>Team{{/link-to}}</li>
+            {{!<li><a href="#"><i class="icon-idea"></i>Ideas</a></li>}}
+        {{/gh-nav-list}}
     </ul>
     {{#if (gh-user-can session.user 'admin')}}
         <ul class="gh-nav-list gh-nav-settings">
             <li class="gh-nav-list-h">Settings</li>
-            <li>{{#link-to "settings.general" classNames="gh-nav-settings-general"}}<i class="icon-settings"></i>General{{/link-to}}</li>
-            {{!<li><i class="icon-design"></i>Themes</li>}}
-            <li>{{#link-to "settings.navigation" classNames="gh-nav-settings-navigation"}}<i class="icon-compass"></i>Navigation{{/link-to}}</li>
-            <li>{{#link-to "settings.tags" classNames="gh-nav-settings-tags"}}<i class="icon-tag"></i>Tags{{/link-to}}</li>
-            <li>{{#link-to "settings.code-injection" classNames="gh-nav-settings-code-injection"}}<i class="icon-code"></i>Code Injection{{/link-to}}</li>
-            <li>{{#link-to "settings.labs" classNames="gh-nav-settings-labs"}}<i class="icon-apps"></i>Labs{{/link-to}}</li>
+            {{#gh-nav-list onClick="closeMenus"}}
```

Comment:
You're kind of breaking html rules by putting a div inside of a ul. Probably better off just having this thing take the place of the ul itself, right?